client/src/components/common/Questionnaire/MultiChoicesQuestion.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Checkbox } from 'antd';
```

Comment:
Here you will be able to use Group directly in tag.
```suggestion
import { Checkbox: { Group }} from 'antd';
```